src/api/routes/user.ts

```diff
@@ -41,6 +42,14 @@ export default () => {
       const { id } = req.params;
       const command = req.body as UpdateUserCommand;
 
+      if (
+        !accessManagerFactory
+          .forAuthentication(req.authentication)
+          .isLoggedInAsUser(new UserId(id))
+      ) {
+        throw new ApiError(404, 'user.not-found');
+      }
+
```

Comment:
Wondering if you can encapsulate this code?  You use it twice in this file, and also in the sharingCode route.

Reply:
Good point. I'll merge and do the refactor with the next PR on access management 👌